packages/earth-admin/src/pages-client/user-profile/home.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useContext } from 'react';
+
+import { Auth0Context } from 'utils/contexts';
+
+import { UserProfileLayout } from 'layouts';
+import { InlineEditCard } from 'components/inline-edit-card';
+import { Input } from 'components/input';
+
+export function UserProfileHome(props: any) {
+  const { userData } = useContext(Auth0Context);
+
+  return (
+    <UserProfileLayout className="marapp-qa-user-profile">
+      <h1 className="ng-text-display-m ng-margin-medium-bottom ng-text-center">Manage your account</h1>
+      <div className="ng-grid">
```

Comment:
pls add ng-form ng-form-dark on a container to have the dark theme form controls